src/main/java/de/berlin/lostberlin/controller/OrderController.java

```diff
@@ -0,0 +1,68 @@
+package de.berlin.lostberlin.controller;
+
+import de.berlin.lostberlin.service.MailService;
+import de.berlin.lostberlin.exception.ResourceNotFoundException;
+import de.berlin.lostberlin.model.Order;
+import de.berlin.lostberlin.repository.OrderRepository;
+import org.apache.commons.validator.routines.EmailValidator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+public class OrderController {
+
+    @Autowired
+    private OrderRepository orderRepo;
+
+    @GetMapping("/orders/all")
+    public List<Order> getAllOrders(){
```

Comment:
It is probably enough to say /orders to get all the orders.

Reply:
We need this method for administrative purposes: get all orders independently from business id. We  will have different access rights for this method, therefore it might be better to keep a separate endpoint.